app/views/content/explore-teaching-advisers/_article.html.erb

```diff
@@ -30,7 +30,7 @@
     </p>
   </section>
 
-  <%= render SubjectSpecific::TextBlockWithTwoImagesComponent.new(
+  <%= render Campaign::TextBlockWithTwoImagesComponent.new(
     heading: "What kind of teacher could you be?",
```

Comment:
On prod, there is a bit more white space above this section / below the body copy which would be good to adjust if possible - gives a bit more breathing space to the page

(this is just one example - this is generally true throughout these pages)

![image](https://user-images.githubusercontent.com/87642394/196925690-e3a280fb-ac89-4a20-9cf0-a1a0b88ef22d.png)

![image](https://user-images.githubusercontent.com/87642394/196925937-be4029ca-97ae-4fb6-a126-15d65ebf1b1f.png)

Reply:
looks much better now